driver/flambda_backend_flags.ml

```diff
@@ -44,6 +44,14 @@ type checkmach_details_cutoff =
 let default_checkmach_details_cutoff = At_most 20
 let checkmach_details_cutoff = ref default_checkmach_details_cutoff
                                        (* -checkmach-details-cutoff n *)
+type checkmach_join =
+  | Keep_all
+  | Widen of int  (* n > 0 *)
+  | Error of int (* n > 0 *)
```

Comment:
`n < 0`? (See https://github.com/ocaml-flambda/flambda-backend/pull/2490/commits/c941ead98a9287947a9665d0fecaefa8484eec4c#diff-39b6a9debf276ee2e6de92522cd7ae1c715822b544c6c38f6a890841eaee49ddR952)

Reply:
Fixed. Parsing of args was wrong in one place, the intent is for the argument to be positive (that's how it is used in `Checkmach`).